test/functional/test_ivn.py

```diff
@@ -93,23 +98,38 @@ def _start_node(self, node_info):
         fake_node_obj.init()
         fake_node_obj.precheck()
         fake_node_obj.start()
+        fake_node_obj.wait_node_up(timeout=10)
```

Comment:
Is 10 seconds enough?
Is this a hardcoded TO value? Or wait_node_up() is going to check node up status intellectually?